lib/crypto/dsa_test.go

```diff
@@ -0,0 +1,28 @@
+package crypto
+
+import (
+  "crypto/dsa"
+  "crypto/rand"
+  "testing"
+)
+
+
+func TestDSA(t *testing.T) {
```

Comment:
I have go test ./... in my pre-commit git hook, go's unit testing in the stdlib is amazing vs other language's stdlib unit testing and I see little reason to use anything else.